### DISCUSSION TITLE
Real usage on desktop and smartphone

### DISCUSSION THREAD

**Discussion by timsofteng** [FiloSottile/age#405] [General]
Hello.
Any existed crossplatforms password managers which use `age` backend?
I know only `gopass` which has `age` as experimental backend.
Unfortunately I can't find solutions for Android or iOS.

**Comment by lucor** [comment]
You could also have a look at `Paw` (https://github.com/lucor/paw) that uses `age` as encryption library.
Currently provides a GUI and CLI apps for linux, macOS, Windows, BSD. Mobile / Web support is planned but not available yet.

Disclaimer: I'm the author of Paw

**Reply by timsofteng** [reply]
> You could also have a look at `Paw` (https://github.com/lucor/paw) that uses `age` as encryption library. Currently provides a GUI and CLI apps for linux, macOS, Windows, BSD. Mobile / Web support is planned but not available yet.
> 
> Disclaimer: I'm the author of Paw

Cool! Looks promising. Unfortunately it still unusable for me without ios/android app, but nevertheless it's cool.
